src/test/java/CourierNegativeLoginTests.java

```diff
@@ -0,0 +1,34 @@
+import io.restassured.response.ValidatableResponse;
+import org.junit.Test;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.junit.Assert.assertThat;
+
+public class CourierNegativeLoginTests {
+
+    public CourierClient courierClient = new CourierClient();
+
+    @Test
+    public void courierNegativeLoginTest(){
+        Courier courierWithoutLogin = new Courier(null, "F4G5H6", "123");
+        ValidatableResponse response = courierClient.login(CourierCredentials.getCourierCredentials(courierWithoutLogin));
+
+        int statusCode = response.extract().statusCode();
+        String errorMessage = response.extract().path("message");
+
+        assertThat("message error","Недостаточно данных для входа", equalTo(errorMessage));
+        assertThat("code error",statusCode , equalTo(400));
+    }
+
+    @Test
+    public void courierNegativePasswordTest(){
```

Comment:
Твой тест падает с очень не хорошей ошибкой:
_Cannot invoke the path method because no content-type was present in the response and no default parser has been set.
You can specify a default parser using e.g.:
RestAssured.defaultParser = Parser.JSON;_
Со стороны это выглядит как косяк тестов, а не косяк сервиса. А это не так. Такая ситуация происходит из-за того, что наш сервер виснет не отдает тело ответа, но зато он отдает статус код. Если проверка на статус код делалась бы раньше проверки body, то тест бы падал с красивой ошибкой статус кода и вопросов бы не было. 
Поэтому я всегда рекомендую первым делом проверять статус код, а потом уже все остальное.

Reply:
сделал.